common/pulp_docker/common/constants.py

```diff
@@ -1,4 +1,7 @@
 IMAGE_TYPE_ID = 'docker_image'
+BLOB_TYPE_ID = 'docker_blob'
+MANIFEST_TYPE_ID = 'docker_manifest'
```

Comment:
I've made a very similar change in a PR against master, but it's slightly different so it will cause a merge conflict:

https://github.com/pulp/pulp_docker/pull/111/files

Can you make your lines the same so it will merge forward cleanly?